CONTRIBUTING.md

```diff
@@ -17,6 +17,7 @@
 
 ## Workflow
 
+* [Sign the CLA](https://cla.shopify.com/)
```

Comment:
Oh, the CLA isn't signed per-repo.  In that case, they might have done so for another repo, in which case they won't need to do that again.

```suggestion
* [Sign the CLA](https://cla.shopify.com/) if you haven't already
```